src/applications/static-pages/sass/modules/_m-facilities.scss

```diff
@@ -191,11 +191,21 @@ $duo-tone-lighten: #000000;
 }
 
 .expand-image-button {
-  background-color: #ffffff;
+  background-color: $color-primary;
```

Comment:
i tried using `class="usa-button"` but the width would not behave correctly - it added a bit of width any way i sliced it